terraform/account/terraform.tfvars.json

```diff
@@ -10,21 +10,25 @@
       "is_production": "false",
       "front_certificate_domain_name": "*.front.lpa.opg.service.justice.gov.uk",
       "admin_certificate_domain_name": "*.admin.lpa.opg.service.justice.gov.uk",
-      "retention_in_days": 7
+      "retention_in_days": 7,
+      "stack_name" : "dev-vpc"
```

Comment:
Given how it's used, I don't think this is necessary. If a namespacing variable was needed account_name.

Reply:
`stack_name` is definitely a hangover from Sirius. You can engineer this out with little effort

Reply:
will remove, and adjust.